lib/features/composer/presentation/composer_controller.dart

```diff
@@ -296,6 +300,9 @@ class ComposerController extends BaseController {
     super.handleExceptionAction(failure: failure, exception: exception);
     if (failure is GetAllIdentitiesFailure) {
       _handleGetAllIdentitiesFailure(failure);
+      if (isSendEmailLoading.value == true) {
+        isSendEmailLoading.value = false;
+      }
```

Comment:
IMO, We don't need loading sending button here, GetAllIdentities is only called when initializing composer.